.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
This file is causing the PnP files to be deleted, which will stop us from the zero-installs feature.

Reply:
Reverted